tests/std/tests/VSO_0000000_instantiate_iterators_misc/test.compile.pass.cpp

```diff
@@ -571,18 +571,24 @@ void future_test() {
 }
 #endif // _M_CEE_PURE
 
-template <typename IoManipIn, typename IoManipOut>
-void iomanip_test_impl(IoManipIn in, IoManipOut out) {
+template <typename IoManipOut, typename IoManipIn>
+void iomanip_test_impl(IoManipOut out, IoManipIn in) {
     stringstream ss{};
-    ss << in;
-    ss >> out;
+    ss << out;
+    ss >> in;
 }
 
 template <typename IoManip>
 void iomanip_test_impl(IoManip iom) {
     iomanip_test_impl(iom, iom);
 }
 
+template <typename IoManip>
+void iomanip_test_impl_for_setfill(IoManip out) {

```

Comment:
It's a little weird that this generic function is used only with `setfill` and includes `setfill` in its name. I'd call this `_output_only` or something, describing what it does instead of how it's currently used. This isn't worth resetting testing, though, so don't bother if we don't need to touch something else.